README.md

```diff
@@ -80,19 +80,19 @@ xcodebuild test \
   -workspace wire-ios-mono.xcworkspace \
   -scheme Wire-iOS \
   -testPlan SecurityTests \
-  -destination 'platform=iOS Simulator,name=iPhone 8,OS=15.2'
+  -destination 'platform=iOS Simulator,name=iPhone 14,OS=17.0.1'
 
   xcodebuild test \
   -workspace wire-ios-mono.xcworkspace \
   -scheme WireSyncEngine \
   -testPlan SecurityTests \
-  -destination 'platform=iOS Simulator,name=iPhone 8,OS=15.2'
+  -destination 'platform=iOS Simulator,name=iPhone 14,OS=17.0.1'
 
   xcodebuild test \
   -workspace wire-ios-mono.xcworkspace \
   -scheme WireDataModel \
   -testPlan SecurityTests \
-  -destination 'platform=iOS Simulator,name=iPhone 8,OS=15.2'
+  -destination 'platform=iOS Simulator,name=iPhone 14,OS=17.0.1'
```

Comment:
Since we recently changed the default device (see [fastlane/.env](https://github.com/wireapp/wire-ios/blob/develop/fastlane/.env)) I adjusted the readme as well.